### PR TITLE
paramaterize centrality tests

### PR DESCRIPTION
**Pull request by birm**
re #1184

This attempts to organize/parameterize tests in centrality/*

### CODE REVIEW

File: test/centrality/betweenness.jl [hunk 1]
```diff
@@ -1,74 +1,77 @@
...
-    for g in testdigraphs(a2)
-        distmx2 = [Inf 2.0 Inf; 3.2 Inf 4.2; 5.5 6.1 Inf]
-        c2 = [0.24390243902439027,0.27027027027027023,0.1724137931034483]
-        @test isapprox(betweenness_centrality(g, vertices(g), distmx2; normalize=false), [0.0,1.0,0.0])
-        @test isapprox(betweenness_centrality(g, vertices(g), distmx2; normalize=false, endpoints=true), [4.0,5.0,4.0])
-        @test isapprox(betweenness_centrality(g, vertices(g), distmx2; normalize=true), [0.0,0.5,0.0])
-        @test isapprox(betweenness_centrality(g, vertices(g), distmx2; normalize=true, endpoints=true), [2.0,2.5,2.0])
+        adjmx2 = [0 1 0; 1 0 1; 1 1 0] # digraph
+        a2 = SimpleDiGraph(adjmx2)
+        for g in testdigraphs(a2)
```
Comment (simonschoelly) [inline test/centrality/betweenness.jl]: Should we parametrize this too?

File: test/centrality/closeness.jl [hunk 2]
```diff
@@ -1,41 +1,48 @@
 @testset "Closeness" begin
-    g5 = SimpleDiGraph(4)
-    add_edge!(g5, 1, 2); add_edge!(g5, 2, 3); add_edge!(g5, 1, 3); add_edge!(g5, 3, 4)
-
-    for g in testdigraphs(g5)
-        y = @inferred(closeness_centrality(g; normalize=false))
-        z = @inferred(closeness_centrality(g))
-        @test y == [0.75, 0.6666666666666666, 1.0, 0.0]
-        @test z == [0.75, 0.4444444444444444, 0.3333333333333333, 0.0]
+    @testset "basic digraph" begin
```
Comment (simonschoelly) [inline test/centrality/closeness.jl]: Not sure if `basic` it the correct word to use here.

File: test/centrality/closeness.jl [hunk 3]
```diff
@@ -1,41 +1,48 @@
...
+        end
     end
 
-    g5 = SimpleGraph(5)
-    add_edge!(g5, 1, 2)
-    for g in testgraphs(g5)
-        z = @inferred(closeness_centrality(g))
-        @test z[1] == z[2] == 0.25
-        @test z[3] == z[4] == z[5] == 0.0
+    @testset "basic undirected" begin
```
Comment (simonschoelly) [inline test/centrality/closeness.jl]: see above

File: test/centrality/eigenvector.jl [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
-    g1 = smallgraph(:house)
-    g2 = cycle_digraph(4)
-
-    for g in testgraphs(g1)
-        y = @inferred(eigenvector_centrality(g))
-        @test round.(y, digits=3) == round.([
-            0.3577513877490464, 0.3577513877490464, 0.5298987782873977,
-            0.5298987782873977, 0.4271328349194304
-        ], digits=3)
+    @testset "smallgraph" begin
```
Comment (simonschoelly) [inline test/centrality/eigenvector.jl]: Maybe rather call this testset "house graph"

File: test/centrality/pagerank.jl [hunk 5]
```diff
@@ -18,15 +18,15 @@
     add_edge!(g6, 1, 2); add_edge!(g6, 2, 3); add_edge!(g6, 1, 3); add_edge!(g6, 3, 4)
     for α in [0.75, 0.85]
 
-        for g in testdigraphs(g5)
+        @testset "$g w/ $α iterations" for g in testdigraphs(g5)
```
Comment (simonschoelly) [inline test/centrality/pagerank.jl]: Alpha is not the number of iterations, but the the the damping factor.

**Comment by sbromberger** [comment]
I'm about to dump a whole slew of changes to the centrality measures. If we merge this, there's a chance it'll make it into the new Centrality framework.

**Pull request merged**